src/com_tjvendors/admin/models/vendors.php

```diff
@@ -270,9 +271,8 @@ public function deleteClientFromVendor($vendor_id, $client)
 			$this->deleteVendor($vendor_id);
 		}
 
-		$dispatcher = JDispatcher::getInstance();
 		PluginHelper::importPlugin('tjvendors');
-		$dispatcher->trigger('tjvendorOnAfterVendorDelete', array($vendorData, $client));
+		Factory::getApplication()->triggerEvent('tjvendorOnAfterVendorDelete', array($vendorData, $client));
```

Comment:
dont triggers needs to be starting as `on`

Reply:
done